src/components/EventCreationPage/FormField.tsx

```diff
@@ -39,17 +39,20 @@ const FormField = ({
   onChange,
   value,
   handleFocus,
+  error,
 }: {
   label: string;
   placeholder: string;
   onChange: (e: any) => any;
-  value: string;
+  value: string | number | null;
   handleFocus?: () => any;
```

Comment:
we can move this file into common components since they're to just being used for the event creation page

Reply:
aren't we still using `FormField` in the event creation page? See here: https://github.com/uwblueprint/paramedics-react/blob/6174b0fda1fa391efca8d73f4f47b41f0d089fc4/src/containers/EventCreationPage.tsx#L110

Reply:
not only being used for event creation*, we can put it in the common folder right?

Reply:
It seems like `src/components/EventCreationPage/FormFields.tsx` is only being used in the EventCreationPage container. There is already a common FormFields component in the common components folder. That being said, @JasonYG do you know the difference between the EventCreationPage/FormField.tsx vs common/FormField.tsx

Reply:
Yeah I mean if they're essentially the same we probably only need one